app/controllers/api/courses/repo_commit_events_controller.rb

```diff
@@ -0,0 +1,18 @@
+module Api::Courses
+    class RepoCommitEventsController < ApplicationController
+        respond_to :json
+        before_action :load_parent
+        load_and_authorize_resource :github_repo, through: :course
+        load_and_authorize_resource :repo_commit_event, through: :github_repo
+
+        def index      
+            respond_to do |format|
+                format.json { render json: @github_repo.repo_commit_events }
```

Comment:
Note: In a future issue, we might want to either filter the commit events based on whether the method `  def self.commit_meets_inclusion_criteria?(repo, c)` from the file `app/models/github_repo.rb` returns true, or possibly calculate that value and include it in the JSON.

The purpose of that would be to enable the visualizations based on this API to match the inclusions/exclusion criteria we are exporting in the CSV files used for research.